simapp/simd/cmd/root.go

```diff
@@ -207,6 +200,29 @@ func addModuleInitFlags(startCmd *cobra.Command) {
 	crisis.AddModuleInitFlags(startCmd)
 }
 
+func genesisCommand(encodingConfig params.EncodingConfig) *cobra.Command {
```

Comment:
it think we should instead add a helper function that adds this, in which users can add more commands to it. This would help avoid extra boiler plate to user files.